roles/convert/tasks/main.yml

```diff
@@ -32,4 +36,8 @@
   ansible.builtin.file:
     name: "{{ convert2rhel_configfile.path }}"
     state: absent
+
+- name: Display Conversion Status
+  ansible.builtin.debug:
+    msg: "Conversion was {{ 'inhibited' if ansible_facts.conversion_inhibited else 'successful' }}"
```

Comment:
Maybe call it before the task with reboot? (I think the reboot should be last in any case)

:thinking:  the role import is fine, that should work.. just thinking what happens if the report to be parsed do not exist before the actual convert task is called, will it actually fail or will it be like silent fail and the play will continue

Reply:
Yeah, please, moving it to happen before the reboot would be ideal. @solumath